tests/ens/test_setup_address.py

```diff
@@ -63,27 +57,19 @@
             'lots.of.subdomains.tester.eth',
             '0x0d62a759aa1f1c9680de8603a12a5eb175cd1bfa79426229868eba99f4dce692',
         ),
-        (
-            'lots.of.subdomains.tester',
-            'lots.of.subdomains.tester.eth',
-            '0x0d62a759aa1f1c9680de8603a12a5eb175cd1bfa79426229868eba99f4dce692',
-        ),
     ],
 )
 def test_set_address(ens, name, full_name, namehash_hex, TEST_ADDRESS):
     assert ens.address(name) is None
-    owner = ens.owner('tester')
+    owner = ens.owner('tester.eth')
 
     ens.setup_address(name, TEST_ADDRESS)
     assert is_same_address(ens.address(name), TEST_ADDRESS)
 
-    # check that .eth is only appended if guess_tld is True
     namehash = Web3.toBytes(hexstr=namehash_hex)
     normal_name = ens.nameprep(full_name)
     if ens.nameprep(name) == normal_name:
```

Comment:
Can either drop this `if` test, or have an `else: assert False, "all names must be full names"` below.